content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,185 @@ description: >
+
+* Virtual ethernet link ([veth](https://man7.org/linux/man-pages/man4/veth.4.html)) - Virtual link allowing you to convey packets between namespaces. 
+
+* [Network namespace](https://man7.org/linux/man-pages/man7/network_namespaces.7.html) - Form of isolation where resources share a network stack, memory, processing and so on.
+
+* CNI - Container network interface providing network configuration, IP address allocation and possibly CNI-specific methods for packet forwarding that enables inter-pod networking. 
+
+
+## Architecture components
+
+The Kubernetes network model introduces a flexible architecture and accompanying components. You can configure and deploy a combination of these different components to assemble an implementation to best meet your cluster networking requirements. 
+
+Figure 1 illustrates the Kubernetes network architecture and accompanying components used in the examples below. 
+
+{{< figure src="/docs/images/k8net-net-arch2.drawio.svg" alt="k8s net arch" class="diagram-large" caption="Figure 1. K8s Network Architecture and Components" >}}
+
+The network architecture and components consist of the following:
+
+* Nodes configured in virtual (VM) or physical (bare-metal) environments.
+  
+* Pods configured on each node with one or more containers.
+  
+* Each Pod has its own IP address (Pod IP) ; this is per [_address family_](https://www.iana.org/assignments/address-family-numbers/address-family-numbers.xhtml), so a cluster that uses IPv4 and IPv6 networking assigns one IPv4 address to each Pod, and also assigns one IPv6 address to each Pod.
+  
+  * Pods run in their own [network namespace](https://man7.org/linux/man-pages/man7/network_namespaces.7.html). All the containers in a pod share this
+    network namespace. (Network namespaces are not the same as the Kubernetes
+    {{< glossary_tooltip text="namespace" term_id="namespace" >}} concept).
+    
+  * You can assign an Ethernet mac address to each Pod.
+    Kubernetes does not require that a Pod has a unique identity at the data-link layer.
+ 
+* Pods use a virtual point-to-point veth "link" between the pod network namespace and root network namespace. This allows pod packets to utilize network functions defined in the root network namespace.  
+  
+* L2bridge is a virtual L2 bridge that allows attachment, configuration and communications between pods on the same node. Depending on your deployment of container runtimes and CNI plugins, this bridge function could be implemented within the CNI plugin itself.  
+
+* [Network plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#:~:text=A%20CNI%20plugin%20is%20required,releases%20of%20the%20CNI%20specification.) allow pods to communicate even when the source pod destination pod are running on different nodes. Network plugins achieve this in different ways. Networks plugins are also known as _CNI plugins_.
```

Comment:
```suggestion
* [Network plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) allow pods to communicate even when the source pod destination pod are running on different nodes. Network plugins achieve this in different ways. Networks plugins are also known as _CNI plugins_.
```
Is it necessary to point to something specific? What if the content of the article changes? I think it's enough to point directly to the article.